.spi.yml

```
@@ -0,0 +1,12 @@
+version: 1
+builder:
+  configs:
+    - platform: ios
+      scheme: Navigator
+    - platform: macos-xcodebuild
+      scheme: Navigator
+    - platform: tvos
+      scheme: Navigator
+    - platform: watchos
+      scheme: Navigator
+    - documentation_targets: [Navigator]
```

This config isn't quite valid (you can check it here: https://swiftpackageindex.com/validate-spi-manifest).

I would use the following instead:

```suggestion
version: 1
builder:
  configs:
    - scheme: Navigator
      documentation_targets: [Navigator]
```

---

Or, better put, it is valid but it won't actually use the `documentation_targets` key.

---

Meh, I'm getting confused by our own syntax 😅. I'd recommend using the variant I'm proposing at it's shorter and easier to reason about.